packages/rum-core/src/domain/rumEventsCollection/action/trackClickActions.spec.ts

```diff
@@ -442,7 +443,14 @@ describe('trackClickActions', () => {
     clickActionDuration: number = BEFORE_PAGE_ACTIVITY_VALIDATION_DELAY
   ) {
     emulateClickWithoutActivity(clock, target)
-    clock.tick(clickActionDuration)
+    if (clickActionDuration < 0) {
+      // Manually change last click timestamp because negative clock tick does not work since jasmine 4: https://github.com/jasmine/jasmine/pull/1948
+      lastClick.startClocks.timeStamp = getTimeStamp(
+        (BEFORE_PAGE_ACTIVITY_VALIDATION_DELAY - clickActionDuration) as RelativeTime
+      )
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏this is a bit unfortunate. I have a workaround that seems to work without needing to adjust our lib code: https://gist.github.com/BenoitZugmeyer/830eef9a0ebd7281bee8c12e3d36529f